roles/kubernetes_monitoring/defaults/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+##########################
+# operator
+
+operator_user: dragon
+operator_group: "{{ operator_user }}"
+
+##########################
+# kubernetes_monitoring
+
+kubernetes_monitoring_chart_source: "dnationcloud/dnation-kubernetes-monitoring-stack"
+kubernetes_monitoring_chart_version: "3.5.0"
```

Comment:
Add `# renovate datasource=github-releases depName= dnationcloud/dnation-kubernetes-monitoring-stack)` in the line before this line.

Reply:
see 15b597fe13a1c91db5b87a72f90c44fe681698b2